modules/nodes-scheduler-node-selectors-cluster.adoc

```diff
@@ -137,13 +137,13 @@ After making this change, wait for the pods in the `openshift-kube-apiserver` pr
 
 * Use a `MachineSet` object to add labels to nodes managed by the machine set when a node is created:
 
-.. Run the following command to add a node selector to a `MachineSet` object:
+.. Run the following command to add a label to a `MachineSet` object:
```

Comment:
I thinkd it's better to replace "a label" with "labels", for there are two pairs of <key, value>